README.md

```diff
@@ -131,40 +163,44 @@ We use [Cypress](https://www.cypress.io/) for our end-to-end tests. There are a
 * `ng e2e --watch` builds and serves the client but just opens Cypress for you to be able to run the tests you want without closing automatically.
   * This is the same as running `ng serve` and `npm run cy:open` (or `npx cypress open`) at the same time. If you are already running `ng serve` it will be easier to do this rather than closing it and running `ng e2e`.
 
-The main page of Cypress looks like this: 
+> When Nic ran the current version of Cypress for the first time popped up a
+> (macOS) "validation" check. This timed out and failed the first time, but when
+> he ran it again it worked fine the second time.
+
+The main page of Cypress looks like this:
 
 ![image](https://user-images.githubusercontent.com/1300395/107994604-9af40300-6fa2-11eb-9caf-35d804281929.png)
 
-You can click on any of the integration test files to run their tests or run them all. When you run a set of tests you will 
+You can click on any of the integration test files to run their tests or run them all. When you run a set of tests you will
 
 ![image](https://user-images.githubusercontent.com/1300395/107994642-acd5a600-6fa2-11eb-8b88-1f2879e82848.png)
 
 There are a lot of neat things you can do here like inspect each test and find which selectors to use in the tests you are writing. We encourage you to look through some of the Cypress documentation linked in the "Resources" section below.
 
 ### GitHub Actions
 
-There are three GitHub Actions workflows set up in your repo: 
-- [Server Java](../../actions?query=workflow%3A"Server+Java") - JUnit tests for the server (`gradle-build`)
-- [Client Angular](../../actions?query=workflow%3A"Client+Angular") - Karma tests (`ng-test`) and ESLint linting (`ng-lint`) for the client
-- [End to End](../../actions?query=workflow%3AEnd-to-End) - Cypress tests for end-to-end testing
+There are three GitHub Actions workflows set up in your repo:
+
+* [Server Java](../../actions?query=workflow%3A"Server+Java") - JUnit tests for the server (`gradle-build`)
+* [Client Angular](../../actions?query=workflow%3A"Client+Angular") - Karma tests (`ng-test`) and ESLint linting (`ng-lint`) for the client
+* [End to End](../../actions?query=workflow%3AEnd-to-End) - Cypress tests for end-to-end testing
 
 There are badges above that show the status of these checks on the master branch.
 
 ## Resources
 
 ### Angular
 
-- [Testing (Karma)](https://angular.io/guide/testing)
-- [Routing](https://angular.io/guide/router)
-- [Forms](https://angular.io/guide/forms-overview)
-- [Angular Material](https://material.angular.io/)
-- [What are environments in Angular](https://angular.io/guide/build#configuring-application-environments)
-- [Angular CLI](https://angular.io/cli)
+* [Angular Unit Testing (Karma)](https://angular.io/guide/testing)
+* [Angular Routing](https://angular.io/guide/router)
+* [Angular Forms](https://angular.io/guide/forms-overview)
```

Comment:
I didn't think it was necessary to prefix everything with "Angular" here because its already in the header of the section.

Reply:
I thought about that. We did have "Angular" in front of (or in) several things, and I particularly thought it would be useful to include that in the "Routing" one since that word is used for so many different things. Adding it might help make the page a little more robust if students are CTRL-F searching it for something like "Angular Forms".